index.css

```diff
@@ -0,0 +1,198 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.project-wraper {
+  width: 70%;
+  margin: 20px auto;
+  padding: 10px;
+  height: 90vh;
+  box-shadow: 2px 3px 25px 1px rgba(0, 0, 0, 0.65);
+  -webkit-box-shadow: 2px 3px 25px 1px rgba(0, 0, 0, 0.65);
+  -moz-box-shadow: 2px 3px 25px 1px rgba(0, 0, 0, 0.65);
+  position: relative;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  border: 2px solid black;
+  padding: 6px 8px 8px 8px;
+}
+
+.nav-list {
+  list-style: none;
+  display: flex;
+  gap: 10px;
+}
+
+.nav-list-item {
+  cursor: pointer;
+}
+
+a:hover {
+  color: chocolate;
+}
+
+a:visited {
+  color: chartreuse;
+}
+
+.line-seperator {
+  border-right: 2px solid rgb(112, 110, 110);
+}
+
+.books-header {
+  margin: 15px;
+  text-align: center;
+}
+
+.books {
+  border: 2px solid rgb(90, 86, 86);
+  width: 80%;
+  text-align: center;
+  margin: 0 auto;
+  max-height: 70vh;
+  overflow-y: scroll;
+}
+
+hr {
+  display: none;
+}
+
+.books-form {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  margin: 0 auto;
+}
+
+/* .books-wraper {
+    overflow: scroll;
+  } */
```

Comment:
- [ ] Let's not commit code that we are not using to the production codebase. The reason is that we want our code to remain clean and simple. We can always check on git history if we need something that we removed. Read more about [HTML & CSS best practices here.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)